Sources/StreamChat/Workers/Background/MissingEventsPublisher.swift

```diff
@@ -86,6 +103,10 @@ class MissingEventsPublisher<ExtraData: ExtraDataTypes>: EventWorker {
                     self?.eventNotificationCenter.process(payload.eventPayloads)
                 case let .failure(error):
                     log.error("Internal error: Failed to fetch and reply missing events: \(error)")
+                    if let error = (error as? ClientError)?.underlyingError as? ErrorPayload,
+                       error.statusCode == 400 {
```

Comment:
nit: It would be nice to have this logic encapsulated somehow: `if error.isSyncFailedError { ... }`

Reply:
But it only means that in that context, so I'm not sure it makes sense to generalise it. We can do something like `has400statusCode`